From 6f8bbc0d2943224cd0e09e3c8cefce4e00f11785 Mon Sep 17 00:00:00 2001
From: Paul Menzel <pmenzel+amd-gfx@molgen.mpg.de>
Date: Mon, 8 Jul 2019 13:50:05 +0200
Subject: [PATCH 2841/2940] drm/amdgpu: Print out voltage in DM_PPLIB

This is a cryptographically signed message in MIME format.

From: Paul Menzel <pmenzel@molgen.mpg.de>
Date: Sun, 7 Jul 2019 14:23:08 +0200

As the clock is already logged, also log the voltage.

Change-Id: I4660263fd18f0d4c5e5510d91fd7547908b01dee
Signed-off-by: Paul Menzel <pmenzel+amd-gfx@molgen.mpg.de>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
---
 drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_pp_smu.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_pp_smu.c b/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_pp_smu.c
index 547b772b17e6..e0f1bd32567d 100644
--- a/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_pp_smu.c
+++ b/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_pp_smu.c
@@ -309,7 +309,8 @@ static void pp_to_dc_clock_levels_with_voltage(
 			DC_DECODE_PP_CLOCK_TYPE(dc_clk_type));
 
 	for (i = 0; i < clk_level_info->num_levels; i++) {
-		DRM_INFO("DM_PPLIB:\t %d in kHz\n", pp_clks->data[i].clocks_in_khz);
+		DRM_INFO("DM_PPLIB:\t %d in kHz, %d in mV\n", pp_clks->data[i].clocks_in_khz,
+			 pp_clks->data[i].voltage_in_mv);
 		clk_level_info->data[i].clocks_in_khz = pp_clks->data[i].clocks_in_khz;
 		clk_level_info->data[i].voltage_in_mv = pp_clks->data[i].voltage_in_mv;
 	}
-- 
2.17.1

